src/init.cpp

```diff
@@ -608,11 +608,10 @@ bool AppInit2(ThreadHandlerPtr threads)
         {
             CreatePidFile(GetPidFile(), pid);
 
-            // While this is technically successful we need to return false
-            // in order to shut down the parent process. This can be improved
-            // by either returning an enum or checking if the current process
-            // is a child process.
-            return false;
+            // Now that we are forked we can request a shutdown so the parent
+            // exits while the child lives on.
+            StartShutdown();
+            return true;
```

Comment:
Why nit just exit(0) ?

Reply:
Yeah I thought of that too after the release. That should be the appropriate way to do it.